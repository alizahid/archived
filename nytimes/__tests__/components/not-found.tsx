import React from 'react'
import { create } from 'react-test-renderer'

import NotFound from '../../src/components/not-found'

it('renders', () => {
  const tree = create(<NotFound />)

  expect(tree).toMatchSnapshot()
})
