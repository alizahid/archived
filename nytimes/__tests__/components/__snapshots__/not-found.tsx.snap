// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "#000000",
        "fontFamily": "System",
        "fontSize": 20,
        "fontWeight": "600",
        "letterSpacing": 0.75,
        "lineHeight": 25,
      }
    }
  >
    Not found
  </Text>
</View>
`;
